File: Assignemnt-18/TASK-2/Task1.java
public class Task1 {

    /**
     * Checks if a number is positive, negative, or zero.
     * @param num the number to check
     */
    public static void checkNumber(int num) {

        if (num > 0) {
            System.out.println("The number is positive");
        } else if (num < 0) {
            System.out.println("The number is negative");
        } else {
            System.out.println("The number is zero");
        }
    }


    public static void main(String[] args) {
        // Test with different inputs
        checkNumber(-5);   // Output: The number is negative
        checkNumber(0);    // Output: The number is zero
        checkNumber(7);    // Output: The number is positive
    }
}

File: Assignemnt-18/TASK-2/Task1.py
"""Convert conditional statements from Java to Python."""

def check_number(num):
    """
    Check if a number is positive, negative, or zero.
    
    Args:
        num: An integer to check
        
    Returns:
        A string describing the number's sign
    """
    if num > 0:
        return "The number is positive"
    elif num < 0:
        return "The number is negative"
    else:
        return "The number is zero"


if __name__ == "__main__":
    # Test cases
    test_cases = [-5, 0, 7]
    
    for num in test_cases:
        result = check_number(num)
        print(f"Input: {num} → Output: {result}")